src/test/javascript/spec/component/programming-exercise/programming-exercise.component.spec.ts

```diff
@@ -255,7 +270,7 @@ describe('ProgrammingExercise Management Component', () => {
             comp.toggleAllProgrammingExercises(); // Deselect all
 
             // THEN
-            expect(comp.selectedProgrammingExercises.length).toEqual(0);
+            expect(comp.selectedProgrammingExercises.length).toBe(0);
         });
 
         it('Should check correctly if selected', () => {
```

Comment:
Would you mind replacing the two expects below with `toBe(true)` and `toBe(false)`?